docs/getting-started/networks.md

```diff
@@ -103,12 +104,27 @@ Each `./build/nodeN` directory is mounted to the `/gaiad` directory in each cont
 
 ### Logging
 
-Logs are saved under each `./build/nodeN/gaiad/gaia.log`. Watch them stream in with, for example:
+Logs are saved under each `./build/nodeN/gaiad/gaia.log`. You can also watch logs
+directly via Docker, for example:
 
 ```
-tail -f build/node0/gaiad/gaia.log
+docker logs -f gaiadnode0
 ```
 
+### Keys & Accounts
+
+To interact with `gaiacli` and start querying state or creating txs, you use the
+`gaiacli` directory of any given node as your `home`, for example:
+
+```shell
+gaiacli keys list --home ./build/node0/gaiacli
+```
+
+Now that accounts exists, you may create new accounts and send those accounts
+funds!
+
+**Note**: Each node's seed is located at `./build/<nodeN>/gaiacli/key_seed.json`.
```

Comment:
`<nodeN>` -> `nodeN` to be consistent with the "Logs are saved under each `./build/nodeN/gaiad/gaia.log`"

Reply:
Good catch 👍